src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommand.cs

```diff
@@ -82,55 +89,89 @@ public override int Execute()
                 throw new GracefulException(LocalizableStrings.InstallToolCommandOnlySupportGlobal);
             }
 
-            try
+            if (_configFilePath != null && !File.Exists(_configFilePath))
             {
-                FilePath? configFile = null;
-                if (_configFilePath != null)
-                {
-                    configFile = new FilePath(_configFilePath);
-                }
+                throw new GracefulException(
+                    string.Format(
+                        LocalizableStrings.NuGetConfigurationFileDoesNotExist,
+                        Path.GetFullPath(_configFilePath)));
+            }
 
-                using (var transactionScope = new TransactionScope())
+            // Prevent installation if any version of the package is installed
```

Comment:
nit: just extra a method

Reply:
Clarification: would you like this extracted to a method?